purchase_request/models/purchase_request.py

```diff
@@ -173,6 +172,14 @@ def check_auto_reject(self):
             if not pr.line_ids.filtered(lambda l: l.cancelled is False):
                 pr.write({'state': 'rejected'})
 
+    @api.multi
+    def _subscribe_assigned_to_user(self):
```

Comment:
@zakiuu It would be nice to add the parameter `subtype_ids=None` since it's also supported by `message_subscribe_users`. This parameter will ease the override of this method when you want to subscribe the assigned user to specific subtypes.